platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/FeatureStorage.kt

```diff
@@ -46,27 +47,22 @@ class FeatureStorage {
 
     fun loadFeature(context: Context, fileName: String): Feature {
         val content = ZipFile(File(getFeaturesDir(context), fileName))
-        val manifest = readManifest(context, content)
+        val manifest = readManifest(content)
         return Feature(fileName, content, manifest)
     }
 
     private fun readManifest(
-        context: Context,
         content: ZipFile
-    ): FeatureManifest {
+    ): FeatureManifest? {
         val manifestEntry = content.getEntry("manifest.json")
         if (manifestEntry == null) {
             logger.warn("Missing manifest for '${content.name}'")
-            return FeatureManifest(
-                null, null, null, null, null, null, null, null
-            )
+            return null
         }
         val manifestString =
             content.getInputStream(manifestEntry).reader().readText()
-        return FeatureManifest.parse(
-            manifestString,
-            Language.determine(context)
-        )
+
+        return Core.parseFeatureManifest(manifestString).getOrNull()
```

Comment:
> This just a placeholder for the time being

What happens to language? Before, context was needed to determine the language. Is that done now in the shared code?

Reply:
Yeah, the language is being passed to core upon initialisation (called "bootstrapping")